react-example/src/components/LoginForm.tsx

```diff
@@ -56,60 +71,100 @@ export const LoginForm: FC<Props> = ({ setEmail, logout, refreshJwt }) => {
   };
 
   const handleJwtRefresh = () => {
-    refreshJwt(email);
+    refreshJwt(user);
   };
 
   const handleEditUser = () => {
-    updateEmail(loggedInUser);
+    updateUser(loggedInUser);
     setEditingUser(true);
   };
 
   const handleCancelEditUser = () => {
-    updateEmail('');
+    updateUser('');
     setEditingUser(false);
   };
 
+  const handleRadioChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setUseEmail(e.target.value === 'email');
+  };
+
   const first5 = loggedInUser.substring(0, 5);
   const last9 = loggedInUser.substring(loggedInUser.length - 9);
 
+  const editingUserForm = (
```

Comment:
Since these are basically mini-components, could we use React component formatting for added clarity? 
```suggestion
  const EditingUserForm = () => (
```
and then update it in line 157 to `<EditingUserForm />`

Reply:
Similarly for `userIdOrEmailForm` and other components below